### PR TITLE
Signup

### PR DESCRIPTION
**Pull request by SarahB6**
Created test for signup

### CODE REVIEW

**Review by luke23571113 — COMMENTED** [review]
Successfully runs test, and code looks good. 

============================================= test session starts =============================================
platform darwin -- Python 3.9.6, pytest-7.4.4, pluggy-1.4.0
rootdir: /Users/lukecollins/Documents/Topics/Blocade
plugins: eth-ape-0.7.22, web3-6.18.0
collected 5 items                                                                                             

tests/test_exampleGame.py ...                                                                           [ 60%]
tests/test_mintToken.py .                                                                               [ 80%]
tests/test_signupTester.py .                                                                            [100%]

============================================== 5 passed in 1.45s ==============================================

File: tests/test_exampleGame.py [hunk 1]
```diff
@@ -1,29 +1,29 @@
-from ape import accounts, project
```
Comment (andrewtheiss) [inline tests/test_exampleGame.py:1]: Why is this editing and re-adding the same file content?

**Review by andrewtheiss — CHANGES_REQUESTED** [review]
Files are being changed and re-added / modified without the content of them effectively changing.  Please only update meaningful changes in files as this will mess up anyone who is changing these later.  

The work here (after fixing the requested changes) contains addition of a single setter and a variable.  After changes if it's accepted, this will be good for a B grade.

File: contracts/Signup.vy [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+@external
+def __init__():
+    pass
+
+# Register wallet ID
+@external
+def register_wallet_id(wallet_id: address) -> bool:
+    # Check if the wallet ID is already registered
+    if self.registered_ids[wallet_id]:
+        # Wallet ID is already registered, raise an error
```
Comment (andrewtheiss) [inline contracts/Signup.vy:22]: log not registered

**Review by andrewtheiss — CHANGES_REQUESTED** [review]

**Review by andrewtheiss — APPROVED** [review]

**Review by HarvardWestlake — APPROVED** [review]
Working setter and getter

**Pull request merged**